server/neptune/workflows/internal/deploy/workflow.go

```diff
@@ -200,19 +198,8 @@ func (r *Runner) Run(ctx workflow.Context) error {
 
 		action = OnNotify
 	}
-
-	v := workflow.GetVersion(ctx, AddNotifierVersion, workflow.DefaultVersion, workflow.Version(2))
-
-	var notifierPeriod time.Duration
-	if v == workflow.Version(1) {
-		notifierPeriod = r.NotifierPeriod(ctx, QueueStatusNotifierHourPST)
-	} else if v == workflow.Version(2) {
-		notifierPeriod = r.NotifierPeriod(ctx, QueueStatusNotifierHourUTC)
-	}
-
-	if v > workflow.DefaultVersion {
-		s.AddTimeout(ctx, notifierPeriod, notifyTimerFunc)
-	}
```

Comment:
When deciding to remove this versions, do we first check if their are any running workflows that are still active on the older version, in this case version 1?

Reply:
yes that's correct